ledger.go

```diff
@@ -1667,31 +1622,31 @@ func (l *Ledger) resetTransactionsSyncIndex() {
 }
 
 func (l *Ledger) loadTransactions() error {
-	txNum := 0
+	count := 0
 
 	blocks := l.blocks.Clone()
 	for _, b := range blocks {
-		txs, err := LoadTransactions(l.db, b.Transactions)
+		transactions, err := LoadTransactions(l.db, b.Transactions)
```

Comment:
we already have `transactions` and this variable used in a pretty small blocks, thats why `txs` was used.
same for `txNum`, its much better represents what this counter means than just `count`

Reply:
I think the context for the variable names is already clear in the function name `loadTransactions` which is why I simplified the variable names.